### ISSUE TITLE
Design changes in the new Edit Cell Dialogue

### ISSUE THREAD

**Issue by N-thony** [africanmathsinitiative/R-Instat#8366]
@derekagorhom the new Edit does miss some of our properties while implementing a new dialogue. In the property the `FormBorderStyle` should be `FixedToolWindow` this won't allow user to expand the dialogue by dragging aside. Same time, can you align the controls well and make the space between them consistent? Also reduce the space at the top of the selector. See the screenshot below.
<img width="594" alt="image" src="https://github.com/africanmathsinitiative/R-Instat/assets/68591383/80bdba68-fe56-43f6-8151-3af81b570b78">

**Comment by rdstern** [comment]
@derekagorhom Two more things to fix in the edit cell dialog.  

a) I opened survey and did some edits.  All fine.  Then I opened diamonds from the library and went to edit a cell in about row 300 in carat - the first column.
It gives me an error, because it has "remembered" that I used to be on the survey.  So it is in the wrong place now.  It should be in the right data frame too.
b) When you do Edit cell in row (say 300) it then returns to the top of the page - row 1 - at the end.  I would like it to stay in the same place, so I see the new value.
c) For discussion also with @N-thony I would like to have a shortcut key - sort of like we use <ctrl>C for copy and <ctrl>X and <ctrl>V?  This could be for both edit cell and delete cell.  This seems to be the F2 key for edit cell and <ctrl> D for delete cells?  Though I wonder we could consider <ctrl>E.  That's for entering cell contents in Excel and I am not sure we will need that in R-Instat?
